packages/server/graphql/types/ActionMeeting.ts

```diff
@@ -16,6 +17,43 @@ const ActionMeeting = new GraphQLObjectType<IActionMeeting, GQLContext>({
   description: 'An action meeting',
   fields: () => ({
     ...newMeetingFields(),
+    agendaItem: {
+      type: AgendaItem,
+      description: 'A single agenda item',
+      args: {
+        agendaItemId: {
+          type: GraphQLNonNull(GraphQLID)
+        }
+      },
+      resolve: async ({id: meetingId}, {agendaItemId}, {dataLoader}) => {
+        const agendaItem = await dataLoader.get('agendaItems').load(agendaItemId)
+        if (agendaItem.meetingId !== meetingId) return null
+        return agendaItem
+      }
+    },
+    agendaItemCount: {
+      type: GraphQLNonNull(GraphQLInt),
+      description: 'The number of agenda items generated in the meeting',
+      resolve: async ({agendaItemCount}) => {
+        // only populated after the meeting has been completed (not killed)
+        return agendaItemCount || 0
+      }
+    },
+    agendaItems: {
+      type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(AgendaItem))),
+      description: 'All of the agenda items for the meeting',
+      resolve: async ({id: meetingId}, _args, {dataLoader}) => {
+        return await dataLoader.get('agendaItemsByMeetingId').load(meetingId)
+      }
+    },
+    commentCount: {
+      type: GraphQLNonNull(GraphQLInt),
+      description: 'The number of comments generated in the meeting',
+      resolve: async ({commentCount}) => {
+        // only populated after the meeting has been completed (not killed)
+        return commentCount || 0
```

Comment:
for older checkin meeetings, this is always going to return 0, which may be a bummer.
could you write a migration to determine how many comments & agenda items those? should look similar to https://github.com/ParabolInc/parabol/blob/master/packages/server/database/migrations/20200323143644-retroevents.ts#L21

Reply:
ahh true! I've just added a migration 👍